pages/Documents/ConversationalAI/ConversationOrchestrator/DynamicRouting/advanced-routing.md

```diff
@@ -178,7 +178,7 @@ Recognize when a high value customer (HVC) starts a conversation and immediately
 **Solution:**
 This can be solved using [SDE](engagement-attributes-types-of-engagement-attributes.html), [Dynamic Routing](conversation-orchestrator-dynamic-routing-overview.html), [Conversation Builder](conversation-builder-overview.html), [Context Service](conversation-orchestrator-conversation-context-service-overview.html) and [LivePerson’s internal queue prioritization application](https://knowledge.liveperson.com/contact-center-management-messaging-operations-queue-management-queue-prioritization-overview.html). LivePerson’s queue prioritization tool can host custom business rules to boost queue priority for Conversations that have certain attributes.  For this use-case LivePerson will add a new rule to boost priority of conversations that have value "VIP” for “Customer Type SDE”. (SDEs can be used to capture one/more/multiple customer attributes).
 
-While SDEs are captured directly for conversations originating from web-engagements, they have to be manually set when conversations originate on other channels such as whatsapp or mobile apps. For this use case a routing bot will check for the customers’s VIP status through a CRM (or relevant application) and then update the SDE value for VIP customers. LivePersons Queue prioritization updates queue priority in real time whenever the SDE is updated. Next SDE based routing policies can be configured on Dynamic Routing to route VIP customers to the appropriate skill.
+While SDEs are captured directly for conversations originating from web-engagements, they have to be manually set when conversations originate on other channels such as WhatsApp or mobile apps. For this use case a routing bot will check for the customers’s VIP status through a CRM (or relevant application) and then update the SDE value for VIP customers. LivePersons Queue prioritization updates queue priority in real time whenever the SDE is updated. Next SDE based routing policies can be configured on Dynamic Routing to route VIP customers to the appropriate skill.
```

Comment:
For this use case, **(added a comma)** a routing bot will check for the customers’ **(incorrect apostrophe)** VIP status through a CRM (or relevant application), **(added a comma)** and then update the SDE value for VIP customers. 
LivePersons Queue (is this the name of it or should there be an apostrophe?) It looks grammatically incorrect. >> LivePerson's **or** LivePerson **(but not what is currently there)**